cartography/intel/aws/sagemaker/endpoints.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
```

Comment:
No need to import these. Use the Python 3.9 native typehints intead e.g. instead of doing `Dict[List[str, Any]]` do `dict[list[str, Any]]`.

In older modules I don't comment on this but since this is brand new we might as well do things the slightly cleaner way.